docs/tutorials/services/source-ip.md

```diff
@@ -106,6 +106,7 @@ client_address=10.244.3.8
 command=GET
 ...
 ```
+If client pod and server pod in the same node, the client_address is the client pod IP address. If client pod and server pod in different node, the client_address is the client pod's node flannel IP address.
```

Comment:
Grammar suggestion: "If the client pod and server pod are in the same node, the client_address is the client pod's IP address. However, if the client pod and server pod are in different nodes, the client_address is the client pod's node flannel IP address."